css/style.css

```diff
@@ -247,6 +246,7 @@ hr {
   font-family: 'Inter', sans-serif;
   font-weight: 500;
   line-height: 20px;
+  float: inline-start;
   color: white;
```

Comment:
- [ ] (Optional) The **_About Me_** section of your portfolio has white patches on the right side. It would be nice to make it look very identical to that of the Figma design template you chose by removing the white patches. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/172701494-904d745a-9581-4129-a230-c8db56b533f9.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/172702091-a27123db-48cf-4e4e-8453-4513f6736f3a.png)